### PR TITLE
Replace branded button instead of appending it to avoid duplication

### LINKED ISSUE #1220: Duplicate Google Pay button on Checkout page
Priority: High

#### What I expected

I added the following line in functions.php to add the Payment Request Button to checkout page, as per documentation:

`add_filter( 'wc_stripe_show_payment_request_on_checkout', '__return_true' );`

With the 'Branded' button type, this should display the Google Pay button once on the checkout page.

#### What happened instead

For 'Default' and 'Buy' button types, the button is displayed once as expected, but with the 'Branded' button type, the Google Pay button is displayed twice – one on top of each other. 

Please see this screenshot: https://nimb.ws/Utxe92

#### Steps to reproduce the issue

Add the following line in functions.php to add the Payment Request Button to checkout page, as per documentation:

`add_filter( 'wc_stripe_show_payment_request_on_checkout', '__return_true' );`

Set the option 'Payment Request Button Type' to 'Branded' in plugin settings.

Add an item to cart and go to the Checkout page. The Google Pay button appears twice.

### PR DESCRIPTION
**Pull request by luizreis**
If the payment button is enabled on checkout, the branded button is duplicated every time checkout is re-calculated. This occurs because the button is being appended to the wrapper, instead of replacing its contents. This commit replaces the wrapper contents instead of appending the button, so it's not duplicated.

Fixes #1220.

#### Changes proposed in this Pull Request:

- Replace branded button in wrapper instead of appending it

#### Testing instructions:

1. Add the following filter to add the Payment Request Button to the checkout page:
`add_filter( 'wc_stripe_show_payment_request_on_checkout', '__return_true' );`
2. Set the option 'Payment Request Button Type' to 'Branded' in plugin settings
3. Add an item to the cart and go to the Checkout page
4. The Google Pay button should appear once
5. Toggle a checkout recalculation by changing the billing/shipping country
6. The Google Pay button should appear once

### CODE REVIEW

**Comment by nabeel-s** [comment]
Although I'm not an official reviewer, I can confirm that I have applied the changes in the commit to my site and it works fine on my end.

**Comment by nabeel-s** [comment]
Although I'm not an official reviewer, I can confirm that I have applied the changes in this commit to my site and it works fine on my end.

**Comment by nabeel-s** [comment]
I can confirm that I have applied the changes in this commit to my site and it works fine on my end:

checkout-button

File: assets/js/stripe-payment-request.js [hunk 1]
```diff
@@ -640,7 +640,7 @@ jQuery( function( $ ) {
 				$( '#wc-stripe-payment-request-wrapper, #wc-stripe-payment-request-button-separator' ).show();
 			} else if ( wc_stripe_payment_request.isBrandedPaymentRequestButton( prButton ) ) {
 				$( '#wc-stripe-payment-request-wrapper, #wc-stripe-payment-request-button-separator' ).show();
-				$( '#wc-stripe-payment-request-button' ).append( prButton );
+				$( '#wc-stripe-payment-request-button' ).html( prButton );
```
Comment (v18) [inline assets/js/stripe-payment-request.js:643]: Looks good. I have just one question - from what I can see there can only ever be one button displayed. Is that assumption correct? If so, I think this is good to go.

**Comment by v18** [comment]
@luizreis the change looks good & tests. I added just one question.

File: assets/js/stripe-payment-request.js (hunk 1 above)
Comment (luizreis) [reply assets/js/stripe-payment-request.js:643]: Thanks for the review @v18! Yes, that's right. If there are multiple payment options, they can be selected in the Payment Request API modal once the button is clicked.

**Comment by luizreis** [comment]
> @luizreis the change looks good & tests. I added just one question.

Awesome! I've just pushed an update to address the minified file changes and updated the `.pot` file, which had some changes from another PR.

**Comment by v18** [comment]
🚢

**Pull request merged**